test/e2e/e2e_test.go

```diff
@@ -22,7 +22,25 @@ func TestE2EFlow(t *testing.T) {
 	// full flow from usersignup with approval down to namespaces creation
 	ctx, hostAwait, memberAwait, member2Await := WaitForDeployments(t, &toolchainv1alpha1.UserSignupList{})
 	defer ctx.Cleanup()
-	hostAwait.UpdateToolchainConfig(testconfig.AutomaticApproval().Disabled())
+	defaultMemberConfig := testconfig.NewMemberOperatorConfig(testconfig.MemberStatus().RefreshPeriod("5s"))
+	member2Config := testconfig.NewMemberOperatorConfig(testconfig.MemberStatus().RefreshPeriod("10s"))
+	hostAwait.UpdateToolchainConfig(testconfig.AutomaticApproval().Disabled(), testconfig.Members().Default(defaultMemberConfig.Spec), testconfig.Members().SpecificPerMemberCluster(member2Await.ClusterName, member2Config.Spec))
+
+	t.Run("verify MemberOperatorConfigs synced from ToolchainConfig to member clusters", func(t *testing.T) {
+		t.Run("verify ToolchainConfig has synced status", func(t *testing.T) {
+			VerifyToolchainConfig(t, hostAwait, wait.UntilToolchainConfigHasSyncedStatus(ToolchainConfigSyncComplete()))
+		})
+		// TODO uncomment VerifyMemberOperatorConfig once the serviceaccount issue is resolved:
+		// memberoperatorconfigs.toolchain.dev.openshift.com "config" is forbidden: User "system:serviceaccount:rsenthil-member-operator:e2e-service-account" cannot get resource "memberoperatorconfigs" in API group "toolchain.dev.openshift.com" in the namespace "rsenthil-member-operator"
```

Comment:
I'm not sure why the serviceaccount is missing permissions to get memberoperatorconfig resources. I'll continue digging around on Monday but if anyone can point me to where that's set it'd be much appreciated.

Reply:
Because it's missing here: https://github.com/codeready-toolchain/toolchain-common/blob/master/scripts/add-cluster.sh#L60

Reply:
Hm.. actually it should be in the member client role and it's there: https://github.com/codeready-toolchain/toolchain-common/blob/master/scripts/add-cluster.sh#L90
Not in the host as I said above.

Reply:
The reason is that e2e SA gets the same permissions as the member-operator SA. So, if the SA of the member-operator deployment doesn't have permissions to get that resource, then this SA won't have it either. 

And since there is no controller in member-operator that would require that permission, then it's missing in the ClusterRole: https://github.com/codeready-toolchain/member-operator/blob/master/deploy/cluster_role.yaml
The end-goal is to have separate ClusterRole for the e2e SA with broader privileges easier to manage